doc/src/start_states.md

```diff
@@ -0,0 +1,118 @@
+The following explains the syntax and semantics of Start States in lrlex.<br>
+A working example can be found in the repository at [lrpar/examples/start_states](https://github.com/softdevteam/grmtools/tree/master/lrpar/examples/start_states)
+
+# Motivation
```

Comment:
Minor comment: `#` is really only for the top-level comment in an MD file (so the file should probably start with the line `# Start states`) and then these comments should be `##`.

Reply:
Should be fixed in [3fad471](https://github.com/softdevteam/grmtools/pull/468/commits/3fad4717f20cdb9722ca715e2cb6fb4cec665894)